magento/ece-tools
# MCLOUD-6530: Fix an issue with empty error code in the StepException

### Description
Fixed an issue when `null` error code can be passed into `StepException`.
Added new error code for not installed Elasticsearch in Magento 2.4+

### Fixed Issues (if relevant)
https://jira.corp.magento.com/browse/MCLOUD-6530

### Manual testing scenarios

1. Install Magento 2.4.0 without Elasticsearch service
2. Check that deployment failed with 134 error code

### Contribution checklist
